orchestra/analytics/latency.py

```diff
@@ -0,0 +1,112 @@
+import itertools
+from datetime import timedelta
+from dateutil.parser import parse
+from operator import attrgetter
+from orchestra.models import Task
+from orchestra.project_api.api import get_project_information
+from orchestra.workflow import Step
+from pandas import DataFrame
+
+
+def work_time_df(projects):
+    """
+    Return all projects' task assignment iteration timing information as a
+    dataframe.
+
+    The table schema is:
+        project: the project's ID
+        task: the task's ID
+        assignment: a 0-indexed assignment counter (0 is entry-level, 1 is
+            first review, etc.)
+        iteration: the work iteration for that assignment.
+            - assignment 0 iteration 0 represents the period from task creation
+              to first assignment.
+            - assignment N > 0 iteration 0 represents the period from previous
+              submission to reviewer assignment.
```

Comment:
clarify this documentation:
* iteration 0 represents the period from previous submission to reviewer assignment:
  * If this is the first assignment, 'previous submission' means 'task creation'.
  * Work time will always be 0
  * If this task is auto-assigned, calendar time will also be 0.
* Iterations N > 0 represent the period between worker assignment and worker submission.